### ISSUE TITLE
KEGG annotations

### ISSUE THREAD

**Issue by ay-amityadav** [PacificBiosciences/pb-metagenomics-tools#20]
Hello, 

I'm interested in KEGG annotations for HiFi reads. I'm aware that the MEGAN6 community edition used in `Taxonomic-Functional-Profiling-Protein` workflow doesn't provide these, whereas paid MEGAN6 ultimate edition does. Is there an alternative for this?

Many thanks
Amit

**Comment by dportik** [comment]
Hi Amit,
KEGG requires licensing, and this is built in with the purchase of MEGAN6 UE. I am unfortunately not aware of any simple alternatives for getting KEGG annotations. If you happen to discover an alternative, please post it here.

**Comment by ay-amityadav** [comment]
Thank you @dportik for your reply! For sure, I'll post here if I come across any such alternative.

**Comment by dportik** [comment]
I wanted to provide a quick update. 

I was able to get KEGG annotations using the command line tools from MEGAN-UE, in conjunction with the mapping file for MEGAN-UE. It did not require a license (likely because we aren't querying the KEGG database).

I've created a separate workflow for this in the `Taxonomic-Profiling-Diamond-Megan` repo. To use it, you will need to download MEGAN-UE to obtain the binaries `sam2rma` and `rma2info`, along with the current MEGAN-UE mapping file.
